packages/amplify-graphql-model-transformer/src/__tests__/model-transformer.test.ts

```diff
@@ -41,6 +41,50 @@ describe('ModelTransformer: ', () => {
     parse(out.schema);
   });
 
+  // should successfully transform simple Embeddable type (non-model) schema
+  it('should successfully transform simple Embeddable type (non-model) schema', async () => {
+    const validSchema = `
+      type NonModelType {
+          id: ID!
+          title: String!
+      }
+      `;
+    const transformer = new GraphQLTransform({
+      transformers: [new ModelTransformer()],
+      featureFlags,
+    });
+    const out = transformer.transform(validSchema);
+    expect(out).toBeDefined();
+
+    validateModelSchema(parse(out.schema));
+    parse(out.schema);
+    expect(out.schema).toMatchSnapshot();
+  });
+  // should successfully transform simple non-capitalized Embeddable type (non-model) name schema
+  it('should successfully transform simple non-capitalized Model/Embeddable type (non-model) name schema', async () => {
+    const alsoValidSchema = `
+      type modelType @model {
+          id: ID!
+          title: String!
+          nonModelTypeValue: nonModelType
+      }
+      type nonModelType {
+          id: ID!
+          title: String!
+      }
+      `;
+    const transformer = new GraphQLTransform({
+      transformers: [new ModelTransformer()],
+      featureFlags,
+    });
+    const out = transformer.transform(alsoValidSchema);
+    expect(out).toBeDefined();
+    
+    validateModelSchema(parse(out.schema));
+    parse(out.schema);
+    expect(out.schema).toMatchSnapshot();
```

Comment:
Another useful check to have which specifically covers your fix is something like:
```
expect(out.schema).toContain("input NonModelTypeInput")
```